.azure-pipelines-templates/common.yml

```diff
@@ -22,7 +22,15 @@ jobs:
       - template: coverage.yml
         parameters:
           consensus: ${{ parameters.consensus }}
-    - ${{ if and(eq(parameters.suffix, 'Perf'), not(and(eq(parameters.consensus, 'BFT'), eq(parameters.protocol, 'HTTP')))) }}:
+    - ${{ if and(parameters.push_perf_data, not(and(eq(parameters.consensus, 'BFT'), eq(parameters.protocol, 'HTTP')))) }}:
       - template: push_perf_data.yml
         parameters:
           consensus: ${{ parameters.consensus }}
+    # For now, only CFT builds are released
+    - ${{ if and(eq(parameters.suffix, 'release'), eq(parameters.consensus, 'CFT'))}}:
+      - template: install.yml
+        parameters:
+          protocol: ${{ parameters.protocol }}
+          # For now, HTTP is default install
```

Comment:
Forever!

Reply:
We can remove that altogether once there's only one flavour of CCF (until we release CFT and BFT in different binaries - if we do).